integrations/wallets/nifty.md

```diff
@@ -0,0 +1,99 @@
+
+## Connect Nifty Wallet to Moonbeam
+
+Once you've created an account, you can now connect to the Moonbase Alpha TestNet by creating a custom network. 
+
+Navigate to the settings, in the top left corner, click on the POA dropdown. Scroll down to the bottom and select Custom RPC.
+  
+<img src="/images/nifty/nifty-images-2.png" alt="Create Custom RPC" style="width: 50%; display: block; margin-left: auto; margin-right: auto;" />
+
+Enter the RPC URL for Moonbeam in the New RPC URL Field:
+
+=== "Moonbeam Development Node"
+
+    ```
+      {{ networks.development.rpc_url }}
+    ```
+    
+=== "Moonbase Alpha"
+
+    ```
+      {{ networks.moonbase.rpc_url }}
+    ```
+
+=== "Moonriver"
+
+    ```
+      {{ networks.moonriver.rpc_url }}
+    ```
+
+
+Then click "Save".
+
+<img src="/images/nifty/nifty-images-3.png" alt="Connect to Moonbase Alpha" style="width: 50%; display: block; margin-left: auto; margin-right: auto;" />
+
+The current RPC should change to the Moonbase Alpha RPC URL, and in the top left corner you'll see the network has changed to "Private Network".
+
+<img src="/images/nifty/nifty-images-4.png" alt="Wallet Connected to Moonbase Alpha" style="width: 50%; display: block; margin-left: auto; margin-right: auto;" />
+
+And that is it, you now have Nifty Wallet connected to the Moonbase Alpha TestNet!
+
+## Using Nifty Wallet
+
+Nifty Wallet serves as a Web3 provider in tools such as [Remix](/integrations/remix/). By having Nifty Wallet connected to Moonbase Alpha, you can deploy contracts as you would like using MetaMask, signing the transactions with Nifty instead. 
+
+For example, in Remix, when deploying a smart contract, make sure you select the "Injected Web3" option in the "Environment" menu. If you have Nifty Wallet connected, you will see the TestNet chain ID just below the box (_1287_) and your Nifty Wallet account injected into Remix as well. When sending a transaction, you should see a similar pop-up from Nifty:
+
+<img src="/images/nifty/nifty-images-5.png" alt="Nifty sign transaction" style="width: 50%; display: block; margin-left: auto; margin-right: auto;" />
+
+Ensure you have DEV tokens in your account and, if necessary, head to the faucet to obtain some tokens. By clicking on "Submit," you are signing this transaction, and the contract will be deployed to the Moonbase Alpha TestNet.
+
+!!! note
+    Please note that although your account balance shows ETH, it is not real ETH, and just DEV tokens. 
+
+The transaction will be displayed under the "Sent" tab.
+
+<img src="/images/nifty/nifty-images-6.png" alt="Nifty confirmed transaction" style="width: 50%; display: block; margin-left: auto; margin-right: auto;" />
+
+## Create a New Account
+
+To create a new account, click the user icon in the top right corner and select "Create Account".
+
+<img src="/images/nifty/nifty-images-7.png" alt="Nifty create an account" style="width: 50%; display: block; margin-left: auto; margin-right: auto;" />
+
```

Comment:
I feel that not adding something here is kind of like an abrupt ending to this section. What happens after you click on "Create Account"?